test/test_numpy_interop.py

```diff
@@ -225,6 +225,11 @@ def test_from_numpy(self, device) -> None:
         x.strides = (3,)
         self.assertRaises(ValueError, lambda: torch.from_numpy(x))
 
+    def test_from_list_of_ndarray_warning(self, device):
+        warning_msg = r"Creating a tensor from a list of numpy.ndarrays is extremely slow"
+        with self.maybeWarnsRegex(UserWarning, warning_msg):
```

Comment:
We now have:

https://github.com/pytorch/pytorch/blob/8734e88f0b77cd3480525e41ae8b7d83a7ea4efd/torch/testing/_internal/common_utils.py#L1379

thanks to @mattip, which we should update this to use.